tests/common/test_cgroupapi.py

```diff
@@ -65,24 +65,24 @@ def test_cgroups_should_be_supported_only_on_ubuntu_16_and_later(self):
             (['redhat', '7.7.1908', 'Core'], False),
             (['bigip', '15.0.1', 'Final'], False),
             (['gaia', '273.562', 'R80.30'], False),
-            (['debian' '9.1', ''], False),
+            (['debian' '9.1', ''], False), # pylint: disable=implicit-str-concat,implicit-str-concat-in-sequence
```

Comment:
Nice! 

This is my bad. I just submitted #1986  fixing this. We can remove this suppression.

Reply:
Done!